1-exercises/C-highlight-words/script.js

```diff
@@ -1,5 +1,46 @@
-function highlightWords(paragraph, colours) {
-  // Write your code here...
+const whereToAttach = document.querySelector("#content");
```

Comment:
Well done! Just need notice that a consistent variable naming scheme is essential for code readability.
If HTML elements are named with "El" suffix, then **whereToAttach** shall be renamed